### PR TITLE
Introduce Omaha Recommendation File

### LINKED ISSUE #7: Add recommendation file for Omaha, Nebraska
- [ ] Create a branch
- [ ] Add commit
- [ ] Open a pull request
- [ ] Review
- [ ] Merge

### PR DESCRIPTION
**Pull request by daveriesz**
closes #7

### CODE REVIEW

File: daveriesz-Omaha_Nebraska.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Omaha, Nebraska
+
+### Places to Eat
+- [ ] That place with the good brunch
+- [ ] That other place with the good beer :beers:
```
Comment (mrjonhazlett) [inline daveriesz-Omaha_Nebraska.md:5]: Have you ever been to Tanner's? Good beers and great chicken wings!

File: daveriesz-Omaha_Nebraska.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Omaha, Nebraska
+
+### Places to Eat
```
Comment (kgalliher) [inline daveriesz-Omaha_Nebraska.md:3]: How is the Mexican food in Omaha?

File: daveriesz-Omaha_Nebraska.md (hunk 1 above)
Comment (daveriesz) [reply daveriesz-Omaha_Nebraska.md:5]: added!

File: daveriesz-Omaha_Nebraska.md (hunk 2 above)
Comment (daveriesz) [reply daveriesz-Omaha_Nebraska.md:3]: Never eaten Mexican in Omaha.

**Review by kgalliher — APPROVED** [review]

**Pull request merged**